_src/pattern-library/sass/components/_forms.scss

```diff
@@ -0,0 +1,455 @@
+// ------------------------------
+// edX Pattern Library: Components - Forms
+
+// About: Contains base styling for forms
+
+// #GLOBAL
+// #INPUT TEXT
+// #INPUT RADIO/CHECKBOX
+// #SELECT
+// #TEXTAREA
+// #SLIDERS
+// #PROGRESS
+
+// ----------------------------
+// #GLOBAL
+// ----------------------------
+.fieldset {
+    @include clearfix;
+    margin-bottom: spacing-vertical(mid-small);
+
+    .group-legend {
+        margin-bottom: spacing-vertical(x-small);
+    }
+
+    .field-wrapper {
+        margin-bottom: spacing-vertical(base);
+
+        .helper-text {
+            font-size: font-size(small);
+            color: $field-tip-color;
+        }
+    }
+
+    &.form-group {
+
+        .field-wrapper { // Generally used for radios and checkboxes
+            margin-bottom: spacing-vertical(small);
+        }
+
+        .helper-text { // Generally used for radios and checkboxes
+            margin-top: spacing-vertical(xx-small);
+            margin-left: rem($baseline);
```

Comment:
[RTL](https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#right-to-left-rtl-support), homeskillet.